src/d_main.c

```diff
@@ -1019,10 +1020,10 @@ void IdentifyVersion (void)
               if (i>=10 && !strncasecmp(iwad+i-10,"doom2f.wad",10))
                 {
                   language=french;
-                  puts("DOOM II version, French language");  // killough 8/8/98
+                  I_Printf(VB_INFO, "DOOM II version, French language");  // killough 8/8/98
                 }
               else
-                puts("DOOM II version");
+                I_Printf(VB_INFO, "DOOM II version");
```

Comment:
It prints all the time because the config hasn't loaded yet, right? I think we can print the version information after the config is loaded.